src/viam/sdk/common/utils.cpp

```diff
@@ -135,6 +135,12 @@ void set_client_ctx_authority(grpc::ClientContext& ctx) {
     ctx.set_authority("viam-placeholder");
 }
 
+std::unique_ptr<grpc::ClientContext> make_cc() {
```

Comment:
I'd prefer this not say `cc` and be called `make_client_context`, or `client_context_factory`.

Reply:
Done!